ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/solution_expose.py

```diff
@@ -158,6 +159,12 @@ def chat(bot):
         secret_env=secret_env,
     )
 
+    message = """
+    <h4>Click next to proceed with the payment</h4>
+    Tcp routers are used in the process of being able to expose your solutions. This payment is to deploy a container with a <a target="_blank" href="https://github.com/threefoldtech/tcprouter">tcprouter client</a> on it.
```

Comment:
Link to the tcp router client should have been: https://github.com/threefoldtech/tcprouter#reverse-tunneling

Reply:
okay will update it directly then, thank you